### ISSUE TITLE
Request support for more source images via Config

### ISSUE THREAD

**Issue by cdhunt** [metaparticle-io/package#85]
My motivation is to include support for Windows containers. I _think_ all that's required is making the following values configurable. At least for dotnet.
https://github.com/metaparticle-io/package/blob/89cc3178bac32a56ca4868a8ccde3605e12637ac/dotnet/Metaparticle.Package/Metaparticle.cs#L135-L136

This was mentioned in #53 and #51. I don't think #51 addresses this sufficiently. If I have to maintain a separate Dockerfile, then I lose much of the value of Metaparticle.

**Comment by brendandburns** [comment]
Yeah, I can definitely see introducing the ability to customize the 'FROM' makes a tone of sense.

Want to do the PR?
